satellite/overlay/service.go

```diff
@@ -127,6 +134,14 @@ type UpdateRequest struct {
 	UptimeDQ     float64
 }
 
+// ExitStatusRequest is used to update a node's graceful exit status.
+type ExitStatusRequest struct {
+	NodeID              storj.NodeID
+	ExitInitiatedAt     *time.Time
+	ExitLoopCompletedAt *time.Time
+	ExitFinishedAt      *time.Time
```

Comment:
I don't think the times need to be pointers anymore. You can just check for IsZero

Reply:
Just curious, what's the pros and cons here for using pointers?

Reply:
Since the functions/methods using these values will only be reading them, there really isn't a reason for them to be pointers.